application/models/SurveysInGroup.php

```diff
@@ -73,22 +73,18 @@ public static function getMinimalPermissionRead()
      */
     public function hasPermission($sPermission, $sCRUD = 'read', $iUserID = null)
     {
-        /* If have global for surveys : return true */
-        $sGlobalCRUD = $sCRUD;
-        if (($sCRUD == 'create' || $sCRUD == 'import')) { // Create and import (token, response , question content …) need only allow update surveys
-            $sGlobalCRUD = 'update';
-        }
-        if (($sCRUD == 'delete' && $sPermission != 'survey')) { // Delete (token, response , question content …) need only allow update surveys
```

Comment:
Is it ok to remove this?

Reply:
Yes, 

1. the only permission are 'surveys', and there are a clear right about survey
2. create and import are done after in this commit (and fixed)